frappe/public/js/frappe/form/templates/form_sidebar.html

```diff
@@ -87,6 +87,13 @@
 		<button class="share-doc-btn text-muted btn btn-default icon-btn">
 			<svg class="icon icon-sm"><use href="#icon-add"></use></svg>
 		</button>
+		<span class="avatar avatar-small share-link">
+			<div class="avatar-frame avatar-action">
+				<svg class="icon icon-sm">
+					<use href="#icon-link-url"></use>
+				</svg>
+			</div>
+		</span>
```

Comment:
UI note:

I would like a bit more contrast and same size as the plus button.

![Screenshot-2022-03-10-102948](https://user-images.githubusercontent.com/16315650/157593415-499cec66-9fd0-4762-9f9b-18ac3b5ca7c8.png)

Perhaps this should also say something like **Get Shareable Link** on hover.